mayastor/src/lvs/lvol.rs

```diff
@@ -249,12 +249,12 @@ impl Lvol {
                     name: self.name(),
                 }
             })?;
-            // write zero to the first 8MB which whipes the metadata and the
+            // write zero to the first 8MB which wipes the metadata and the
             // first 4MB of the data partition
             let range =
                 std::cmp::min(self.as_bdev().size_in_bytes(), (1 << 20) * 8);
             debug!(?self, ?range, "zeroing range");
-            for offset in 0 .. (range >> 12) {
+            for offset in 0 .. (range >> 16) {
```

Comment:
Nit: Can we derive this from 8MiB together with the buffer size we're using? To make it easier to change if we ever need to?

Reply:
Sure - done.